### PR TITLE
chore: [TECH-840] migrate getD2

### PR DESCRIPTION
**Pull request by jasminenguyennn**

### CODE REVIEW

**Review by superskip — COMMENTED** [review]
That was one big refactor - great job!

File: src/components/AppLoader/init.js [hunk 1]
```diff
@@ -1,18 +1,18 @@
 // @flow
 /* eslint-disable import/prefer-default-export */
 import log from 'loglevel';
-import { init as initAsync, config } from 'd2';
+import { config } from 'd2';
```
[the remark this answers is not in the document]
Comment (jasminenguyennn) [reply src/components/AppLoader/init.js:4]: @superskip there is another ticket to remove the rest from d2 https://dhis2.atlassian.net/browse/TECH-841

**Review by superskip — APPROVED** [review]
LGTM! :tada:

(Changing paging from false to true still looks like a mistake to me though, but it doesn't matter if that epic is never triggered anyway.)

**Comment by jasminenguyennn** [comment]
> LGTM! 🎉
> 
> (Changing paging from false to true still looks like a mistake to me though, but it doesn't matter if that epic is never triggered anyway.)

thanks @superskip. that must be a merge conflict, I reverted it back to `paging: false`

**Pull request merged**